### PR TITLE
forecast model slides

### PR DESCRIPTION
**Pull request by sbfnk**
missing a picture for statistical models but otherwise consider this to be all we need at this stage

### CODE REVIEW

File: sessions/slides/forecasting-models.qmd [hunk 1]
```diff
@@ -0,0 +1,165 @@
...
+
+[Bosse *et al.*, *Wellcome Open Res*, 2024](https://doi.org/10.12688/wellcomeopenres.19380.2)
+
+:::
+
+::::
+
+# Plan for the this session
+
+1. Try a range of forecasts + evaluate
```
Comment (seabbs) [inline sessions/slides/forecasting-models.qmd:161]:
```suggestion
1. Review the performance of the random walk model from the last session
2. Motivate a mechanism to include in order to address some of the issues we identified
3. Motivate a statistical approach aiming to address the same issues without introducing epidemiological mechanism
4. Compare the performance of these models for a single forecast
5. Evaluate many forecast from these models and compare their performance to the random walk model
```

**Review by seabbs — APPROVED** [review]
LGTM. Suggest we expand the plan. I agree this is all we need. Nice!

**Comment by kathsherratt** [comment]
these look great to me too :)

**Comment by sbfnk** [comment]
any suggestion for a figure for statistical models?

**Comment by seabbs** [comment]
Anything from here work? https://www.pnas.org/doi/full/10.1073/pnas.2111453118?doi=10.1073%2Fpnas.2111453118

**Comment by kathsherratt** [comment]
Or our old friend MUNI-ARIMA
https://www.math.muni.cz/~krausd/covid19stat/covid_pred_dashboard.html?v=2022-01-26

image

or https://krausstat.shinyapps.io/covid19global/

**Pull request merged**